packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/editingcontext/migration/participants/NodeDescriptionLabelExpressionMigrationParticipant.java

```diff
@@ -35,7 +35,7 @@
 @Service
 public class NodeDescriptionLabelExpressionMigrationParticipant  implements IMigrationParticipant {
 
-    private static final String PARTICIPANT_VERSION = "2024.4.0";
+    private static final String PARTICIPANT_VERSION = "2024.3.2.22.0.0";
```

Comment:
I somehow missed that but we discussed in details about the shape that was expected for the migration participant version and I don't understand what this is supposed to be (what is this 22 for example?). I don't know what this is supposed to be. I need an iteration, a date and some time, something like this `2024.8.0-202407121400`. It needs to be more specific in the ADR.

Reply:
I updated the ADR in order to be more precise about the version of participants